### PR TITLE
Hello world!!

### PR DESCRIPTION
**Pull request by p0lygun**
its done 
added my pipinfo script

### CODE REVIEW

**Comment by p0lygun** [comment]
what happened ?

**Comment by JoeNibe** [comment]
@p0lygun you did not lint your code.

File: hello_world/pipinfo.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import json
+import re
+import subprocess
+import sys
+
+
```
Comment (Shivansh-007) [inline hello_world/pipinfo.py:6]: Extra newline here.

File: hello_world/pipinfo.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            f"[red]{pkg['version']}[/red]",
+            f"[#479435]{pkg['latest']}[/#479435]",
+            pkg["path"],
+        )
+    else:
+        table.add_row(*pkg.values())
+
+
+console = Console()
+console.print(table)
```
Comment (JoeNibe) [inline hello_world/pipinfo.py:82]:
```suggestion
console.print(table)

```

Comment (JoeNibe) [reply hello_world/pipinfo.py:82]: i was just checking the review option

Comment (Shivansh-007) [reply hello_world/pipinfo.py:82]: The new line is already present on it, the lint isn't erroring.

GitHub doesn't show the newline, it rather shows if there is no newline present with a red error symbol.

**Review by JoeNibe — APPROVED** [review]
looks good

**Review by JoeNibe — APPROVED** [review]
look good. lets merge it after resolving the newline issue.

File: hello_world/pipinfo.py (hunk 1 above)
Comment (p0lygun) [reply hello_world/pipinfo.py:6]: 2 new lines ?  ok

**Review by p0lygun — COMMENTED** [review]
removed the extra newline

**Pull request merged**